service-registry/src/main/java/io/servicecomb/serviceregistry/api/registry/MicroserviceRegisterWay.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 Huawei Technologies Co., Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.servicecomb.serviceregistry.api.registry;
+
+/**
+ * Created by   on 2017/12/15.
+ */
+public enum MicroserviceRegisterWay {
```

Comment:
just a string, why we need a enum?
some day, add a type, all sdk must update, otherwise will got a unknown?

Reply:
SC's data dictionary shows it's a enum type